src/arsenic/session.py

```diff
@@ -235,6 +235,13 @@ async def execute_script(self, script: str, *args: Any):
             data={"script": script, "args": list(args)},
         )
 
+    async def execute_async_script(self, script: str, *args: Any):
+        return await self._request(
+            url="/execute/async",
+            method="POST",
+            data={"script": script, "args": list(args)},
```

Comment:
`list` is not needed, in the end the whole `data` is dumped as JSON, where both tuples and lists are accepted.